go.mod

```diff
@@ -11,9 +11,10 @@ require (
 	github.com/jessevdk/go-flags v1.4.0
 	github.com/jrick/logrotate v1.0.0
 	github.com/kaspanet/go-muhash v0.0.4
-	github.com/kaspanet/go-secp256k1 v0.0.5
+	github.com/kaspanet/go-secp256k1 v0.0.7
 	github.com/pkg/errors v0.9.1
 	github.com/syndtr/goleveldb v1.0.1-0.20190923125748-758128399b1d
+	github.com/tyler-smith/go-bip39 v1.1.0
```

Comment:
Do we want to clone this to /kaspanet?

Reply:
Don't we plan to clone everything to kaspanet pre mainnet?